FluentUI.Demo/src/main/res/layout/activity_basic_inputs.xml

```diff
@@ -17,7 +17,8 @@
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
         android:paddingBottom="@dimen/demo_headline_padding_bottom"
-        android:text="@string/button" />
+        android:accessibilityHeading="true"
+        android:text="Different views of button" />
```

Comment:
Add "Different views of button" string in String.xml & refer it.

Reply:
Getting that translation error when adding in string